### PR TITLE
:sparkles: show placeholder to label when input is not empty

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: src/group/group.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import classnames from 'classnames'
 import { onlyUpdateForPropTypes } from 'recompose'
 import styles from '../../src/group/group.styles.scss'
 
-const Group = ({ style, className, children, vertical }) => {
+const Group = ({ style, className, children, vertical, withLabel }) => {
```
Comment (fabienjuif) [inline src/group/group.jsx]: Why is this in this PR ?

File: src/input/input.styles.scss [hunk 2]
```diff
@@ -5,9 +5,24 @@
   position: relative;
   width: 15em;
 
+  &.showLabel {
+    label {
+      opacity: 1;
```
Comment (fabienjuif) [inline src/input/input.styles.scss]: This is a bad practice imo

File: src/input/input.jsx [hunk 3]
```diff
@@ -23,13 +23,16 @@ const Input = ({
    className, style,
    type, name, label,
    placeholder, disabled,
-   required, options, value,
+   required, options, value, hiddenLabel,
```
Comment (fabienjuif) [inline src/input/input.jsx]: what about `placeholderToLabel` ?

**Comment by guillaumecrespel** [comment]
2017-10-04_16h09_25

File: src/input/input.jsx (hunk 3 above)
Comment (guillaumecrespel) [reply src/input/input.jsx]: I think the must it's the default value for input not on "Group" it s show placeholder to label.
So i add hiddenLabel only when i'have not need label.

**Pull request merged**